src/main/java/ay2021s1_cs2103_w16_3/finesse/model/ModelManager.java

```diff
@@ -152,6 +182,13 @@ public void setTransaction(Transaction target, Transaction editedTransaction) {
         return FXCollections.unmodifiableObservableList(newFilteredIncomes);
     }
 
+    @Override
+    public ObservableList<FrequentExpense> getFilteredFrequentExpenseList() {
+        ObservableList<FrequentExpense> newFilteredFrequentExpenses = FXCollections.observableArrayList();
+        filteredFrequentExpenses.forEach(i -> newFilteredFrequentExpenses.add(i));
+        return FXCollections.unmodifiableObservableList(newFilteredFrequentExpenses);
+    }
+
```

Comment:
This method seems to be taken from `getFilteredExpenseList` and `getFilteredIncomeList`, which are found to be buggy. Please modify it to be an ordinary getter.